service/dispatch/internal/repo/dispatch_repo.go

```diff
@@ -52,7 +52,7 @@ const getSubscribersOfDispatchQ string = `
 	on cs.dispatch_id = cd.id
 	left join subs."users" u 
 	on cs.user_id = u.id
-	where cd.u_id = $1 and u.email is not null;
+	where cd.u_id = $1 and u.email is not null and cs.status != 2;
```

Comment:
`cs.status != 2` seems magic and not understandable. Use constant with proper name for this, so everybody who is reading your code will understand it imidiatly

Reply:
Refined sql query via usage of subquery

Reply:
Sry. I mean not to use subquery, but just make local constant and use something like Sprintf, so it will be clear what is status '2' means.

Reply:
Oh, gotcha. I've replaced subquery with constant value

Reply:
Cool